config/jobs/kubernetes-sigs/azuredisk-csi-driver/azuredisk-csi-driver-config.yaml

```diff
@@ -360,24 +360,55 @@ presubmits:
       testgrid-tab-name: pr-in-tree-azure-disk-e2e-vmss
       description: "Run Azure Disk e2e test on a VMSS-based cluster with Azure Disk in-tree volume plugin."
       testgrid-num-columns-recent: '30'
-  - name: pull-azuredisk-csi-driver-windows-build
```

Comment:
would you keep `pull-azuredisk-csi-driver-windows-build` and add another e2e pipeline? Thanks.